opencensus/metrics/transport.py

```diff
@@ -61,19 +62,23 @@ def func(*aa, **kw):
             except Exception:
                 logger.exception("Error handling metric export")
 
-        super(MetricExporterTask, self).__init__(interval, func, args, kwargs)
+        super(PeriodicMetricTask, self).__init__(interval, func, args, kwargs)
 
 
-def get_exporter_thread(metric_producer, exporter, interval=None):
+def get_exporter_thread(metric_producers, exporter, interval=None):
     """Get a running task that periodically exports metrics.
 
     Get a `PeriodicTask` that periodically calls:
 
-        exporter.export_metrics(metric_producer.get_metrics())
+        exporter.export_metrics(all_metrics)
```

Comment:
Consider putting the actual code example here (instead of the description below).
Consider `itertools.chain`.